py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -152,10 +159,17 @@ def _get_connection_manager(self):
 
         return urllib3.PoolManager(**pool_manager_init_args)
 
-    def __init__(self, remote_server_addr: str, keep_alive: bool = False, ignore_proxy: bool = False):
-        self.keep_alive = keep_alive
+    def __init__(self, remote_server_addr: str, client_config: ClientConfig = ClientConfig()):
```

Comment:
So in Python, any object that is a function default is "cached", unlike Ruby, where `client_config` will always get a new instance of `ClientConfig`. 

This may affect scripts that run multiple browsers in one session, if they don't set a different default client config.

Reply:
So the better pattern is to use None as default and then set it in the constructor body?